zerver/webhooks/github/view.py

```diff
@@ -283,6 +283,7 @@ def get_push_commits_body(helper: Helper) -> str:
         get_branch_name_from_ref(payload["ref"].tame(check_string)),
         commits_data,
         deleted=payload["deleted"].tame(check_bool),
+        push_type="force-pushed" if payload["forced"].tame(check_bool) else "pushed",
```

Comment:
I would spell this "force pushed", and maybe call the parameter `push_type_description` to make clear it's a user-facing string, not a computer-use token like `force-pushed`.

Actually, what I'd actually do is make `get_push_commits_event_message` support an optional `force_push=True|False`, and be responsible for deciding which string to use based on that (only this integration would pass the `True` value at first). And then it'll be much easier to make other Git integrations support this as follow-ups, which you should open an issue for once this is integrated.

Reply:
Changing the parameter to something like a `boolean` is for sure a good idea, as it would also be easy for others to deduce what this parameter will actually do. 

I will make the suggested change and will also make sure to open follow-ups after this one.